src/Template/Element/advanced_search_form.ctp

```diff
@@ -357,6 +372,18 @@ echo $this->Form->create('AdvancedSearch', [
                    href="http://en.wiki.tatoeba.org/articles/show/text-search">
             <?= __('More search options') ?>
         </md-button>
+        <?php if (!(isset($isSidebar) && $isSidebar)): ?>
+            <md-button type="submit" class="md-primary" formaction="">
+                <?= __('Create a search template') ?>
+            </md-button>
+            <span>
+                <md-icon class="unsure">help</md-icon>
+                <md-tooltip md-direction="right" md-delay="1000">
+                    <?= __('Use this button to use the currently selected criteria as a base for other searches.
+                        You can also bookmark/share the search template with someone else.') ?>
```

Comment:
I’m not exactly sure why, but I have trouble seeing the tooltip. It keeps blinking every second. I think you can reproduce the problem if you shrink your window enough so that the text overflows the browser window.

Another (probably unrelated) problem is that the string contains a "line return" character as well as lots of spaces before the word "You". This is likely to show up strangely for UI translators. You can break the string into multiple chunks using the dot operator to keep a nice indentation like this:
```php
                    <?= __('This is a string '    // notice the final space here
                          .'on multiple lines '   // and here
                          .'nicely indented.') ?>
```

Reply:
Looks like this is partially fixed by changing the tooltip direction to "top". It still runs off the page, but it doesn't flash.

I looked into putting newlines or doing automatic wrapping in md-tooltips and it doesn't seem like they're supported. Would love to be wrong about this though.

Reply:
Good job fixing the blink problem. :+1:

I’ve looked into how to make a multiline tooltip that automatically wraps. I was able to get something working with the following code:
```html
<!-- src/Template/Element/advanced_search_form.ctp -->
<md-tooltip class="multiline" ...>
```

```css
/* webroot/css/sentences/advanced_search.css */
md-tooltip.multiline {
    /* Only wrap tooltip if it takes more than 90% of screen width. */
    max-width: 90%;
}

md-tooltip.multiline .md-content {
    height: auto;
    white-space: normal;
    /* The original md-tooltip use line-height to pad, but we want a nice
     * multi-line text, so we need to use line-height instead. */
    line-height: normal;
    padding: 5px 8px;
}
```

90% is not enough to make the text wrap using the current text length, but you can see the result by lowering this value. This should make the tooltip responsive-proof and long-UI-translations-proof.